static/storage-change-detection.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
Can you duplicate the comment [here](https://github.com/rust-lang/docs.rs/blob/819f5dc8aac441d057b5fe6fdb39d7c1a3c52afe/templates/rustdoc/body.html#L40-L42) so that the purpose of this file is clear?